### PR TITLE
[metalperformanceshaders] Update up to beta 5

### PR DESCRIPTION
**Pull request by spouliot**

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build failure

File: src/metalperformanceshaders.cs [hunk 1]
```diff
@@ -489,4 +540,592 @@ interface MPSImageTranspose {
...
+		MTLRegion ClipRect { get; set; }
+
+		[Export ("destinationFeatureChannelOffset")]
+		nuint DestinationFeatureChannelOffset { get; set; }
+
+		[Export ("edgeMode", ArgumentSemantic.Assign)]
+		MPSImageEdgeMode EdgeMode { get; set; }
+
+		[Export ("encodeToCommandBuffer:sourceImage:destinationImage:")]
+		void EncodeToCommandBuffer (IMTLCommandBuffer commandBuffer, MPSImage sourceImage, MPSImage destinationImage);
```
Comment (VincentDondain) [inline src/metalperformanceshaders.cs]: `EncodeToCommandBuffer` -> `Encode`

Comment (spouliot) [inline src/metalperformanceshaders.cs]: ideally yes, but its too late: older API are using `Encode[Transform]ToCommandBuffer`

File: src/metalperformanceshaders.cs [hunk 2]
```diff
@@ -489,4 +540,592 @@ interface MPSImageTranspose {
...
+
+		[Export ("rightMatrixOrigin", ArgumentSemantic.Assign)]
+		MTLOrigin RightMatrixOrigin { get; set; }
+
+		[DesignatedInitializer]
+		[Export ("initWithDevice:transposeLeft:transposeRight:resultRows:resultColumns:interiorColumns:alpha:beta:")]
+		IntPtr Constructor (IMTLDevice device, bool transposeLeft, bool transposeRight, nuint resultRows, nuint resultColumns, nuint interiorColumns, double alpha, double beta);
+
+		[Export ("encodeToCommandBuffer:leftMatrix:rightMatrix:resultMatrix:")]
+		void EncodeToCommandBuffer (IMTLCommandBuffer commandBuffer, MPSMatrix leftMatrix, MPSMatrix rightMatrix, MPSMatrix resultMatrix);
```
Comment (VincentDondain) [inline src/metalperformanceshaders.cs]: `EncodeToCommandBuffer` -> `Encode`

**Comment by VincentDondain** [comment]
LGTM 👍

**Pull request merged**